appeals/web/src/server/appeals/appeal-documents/appeal-documents.mapper.js

```diff
@@ -586,6 +586,46 @@ function mapDocumentDetailsItemToDocumentDetailsPageComponents(item, redactionSt
 
 	return pageComponents;
 }
+/**
+ * @param {string} fileName
+ * @param {{originalFilename: string, documentId: string}} item
+ * @returns {PageComponent[]}
+ */
+function mapDocumentNameItemToDocumentNamePageComponents(item, fileName) {
+	/** @type {PageComponent[]} */
+	const pageComponents = [
+		{
+			wrapperHtml: {
+				opening: `<div class="govuk-form-group"><h2 class="govuk-heading-m">${item.originalFilename}</h2>`,
+				closing: ''
+			},
+			type: 'input',
+			parameters: {
+				type: 'hidden',
+				name: `documentId`,
```

Comment:
Super pedantic, and it doesn't make any difference really, but there's no need for this to be a template string in this case.

Reply:
Done as requested